src/masoniteorm/schema/platforms/SQLitePlatform.py

```diff
@@ -0,0 +1,143 @@
+
+
+class SQLitePlatform:
+
+    type_map = {
+        "string": "VARCHAR",
+        "char": "CHAR",
+        "integer": "INTEGER",
+        "big_integer": "BIGINT",
+        "tiny_integer": "TINYINT",
+        "big_increments": "BIGINT",
+        "small_integer": "SMALLINT",
+        "medium_integer": "MEDIUMINT",
+        "increments": "INTEGER PRIMARY KEY",
+        "binary": "LONGBLOB",
+        "boolean": "BOOLEAN",
+        "decimal": "DECIMAL",
+        "double": "DOUBLE",
+        "enum": "ENUM",
+        "text": "TEXT",
+        "float": "FLOAT",
+        "geometry": "GEOMETRY",
+        "json": "JSON",
+        "jsonb": "LONGBLOB",
+        "long_text": "LONGTEXT",
+        "point": "POINT",
+        "time": "TIME",
+        "timestamp": "TIMESTAMP",
+        "date": "DATE",
+        "year": "YEAR",
+        "datetime": "DATETIME",
+        "tiny_increments": "TINYINT AUTO_INCREMENT",
+        "unsigned": "INT UNSIGNED",
+        "unsigned_integer": "UNSIGNED INT",
+    }
+
+    def compile_create_sql(self, table):
+        columns = self.columnize(table.get_added_columns())
+        
+        return self.create_format().format(
+            table=self.table_string().format(table=table.name).strip(),
+            columns=', '.join(self.columnize(table.get_added_columns())).strip(),
+            constraints=', ' + ', '.join(self.constraintize(table.get_added_constraints())) if table.get_added_constraints() else ""
+        )
+
+    def compile_alter_sql(self, diff):
```

Comment:
@Marlysson @girardinsamuel I think writing the methods like this has been much easier with the table and table diff structure